config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-ci.yaml

```diff
@@ -0,0 +1,19 @@
+periodics:  
+- interval: 4h
+  name: cluster-api-provider-aws-test-creds
+  always_run: true
+  decorate: true
+  labels:
+    preset-service-account: "true"
+    preset-aws-credential: "true"
+  path_alias: "sigs.k8s.io/cluster-api-provider-aws"
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/cluster-api:v20180720-d52d72975      
```

Comment:
Can you explain what led you to decide to use this container with the execute scenario instead of calling a bare script in the kubekins container?  I honestly don't know which is correct or why atm, I'm wondering if you do

Reply:
> I honestly don't know which is correct or why atm, I'm wondering if you do 

I do not. I grabbed it from `cluster-api-ci.yaml` I'm a bit overwhelmed by the lack of docs, to be honest.

Reply:
It looks like the kubekins image has the aws-cli which should be the only requirement needed for the test script.

Reply:
Let me change it.